sympy/geometry/polygon.py

```diff
@@ -401,6 +401,35 @@ def centroid(self):
             cx += v*(x1 + x2)
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
+ 
+    @property
+    def second_moment_of_area(self):
```

Comment:
This should be:

```python
def second_moment_of_area(self, point=None):
```

The default point can be the centroid, but you should also be able to pass in another point. In beams, which you want to use this for, it is not always true that the neutral axis aligns with the centroid, in the most general sense.

Reply:
ok sir, I will make the changes. the function will find I_xx and I_yy about the centroid and using the parallel axis theorem will find about any point( if needed).
is it good idea to ask the user to enter the points in a counterclockwise manner because if the user enters in a clockwise manner I_xx and I_yy will be negative. It may give the wrong answer using parallel axis theorem (I  = I_c +A*d**2). As everything is symbolic we cant say anything about expression.